plugins/modules/events.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import requests
+
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: events
+
+short_description: Handle AssistedInstall cluster events
+
+version_added: "1.0.0"
+
+description: Interact with cluster events through the AssistedInstall API
+
+options:
+    action:
```

Comment:
@tonyskapunk Do you think we need action for events?

Reply:
I think you could potentially add a new assisted installer event through a [POST](https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L3639) which could in turn mean a new action.